bin/start_kube.sh

```diff
@@ -239,41 +239,59 @@ fi
 fi
 
-if pulumi config get sirius:accounts_pwd -C "${script_dir}"/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
-	echo "Configuration value found"
+# Sirius Ledger Database
+if pulumi config get sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
+  true
 else
-	echo "Please enter a password for the sirius accountsdb"
-	pulumi config set sirius:accounts_pwd --secret -C pulumi/python/kubernetes/secrets
+  LEDGER_PW=$(createpw)
+  pulumi config set --secret sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kubernetes/secrets $LEDGER_PW
 fi
 
-if pulumi config get sirius:demo_login_pwd -C "${script_dir}"/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
-	echo "Configuration value found"
+if pulumi config get sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
+  true
 else
-	echo "Please enter a password for the sirius ledgerdb"
-	pulumi config set sirius:demo_login_pwd --secret -C pulumi/python/kubernetes/secrets
+  LEDGER_PW=$(createpw)
+  pulumi config set --secret sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kubernetes/secrets $LEDGER_PW
 fi
 
-if pulumi config get sirius:demo_login_user -C "${script_dir}"/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
+# Admin password for grafana (see note in __main__.py in prometheus project as to why not encrypted)
+# This is for the deployment that is setup as part of the the prometheus operator driven prometheus-kube-stack.
+#
+if pulumi config get prometheus:adminpass -C ${script_dir}/../pulumi/python/config >/dev/null 2>&1; then
+  echo "Existing password found for grafana admin user"
+else
+  echo "Create a password for the grafana admin user; this password will be used to access the Grafana dashboard"
+  echo "This should be an alphanumeric string without any shell special characters; it is presented in plain text"
+  echo "due to current limitations with Pulumi secrets. You will need this password to access the Grafana dashboard."
+  pulumi config set prometheus:adminpass -C ${script_dir}/../pulumi/python/config
+fi
+
+#
+# The demo account defaults to testuser/password for credentials; this needs to be fixed in the BoS to allow
+# other passwords to be used. Once that is done we can adjust this section.
+#
+if pulumi config get sirius:demo_login_pwd -C "${script_dir}"/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
 	echo "Configuration value found"
 else
-	echo "Please enter a username for the BoS"
-	pulumi config set sirius:demo_login_user --secret -C pulumi/python/kubernetes/secrets
+	pulumi config set sirius:demo_login_pwd --secret -C pulumi/python/kubernetes/secrets "password"
 fi
 
-if pulumi config get sirius:ledger_pwd -C "${script_dir}"/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
+if pulumi config get sirius:demo_login_user -C "${script_dir}"/../pulumi/python/kubernetes/secrets >/dev/null 2>&1; then
```

Comment:
nit: It looks like we'll just get a lot of "configuration value found" printed with no indication of what configuration value we are talking about.  Suggest either removing them or specifying which config key already had a value.

Reply:
Good point; changed it to "true" for the internal passwords, so they won't get any output there. For the demo user, I added logic to say "hey, I found this".